### PR TITLE
Enhancement/withdrawal bridging error message

### PR DESCRIPTION
**Pull request by Aierie**
Addresses CS-1462.

- Adds error UI to the withdrawal status card 
- Adds component tests for the withdrawal status card and the deposit status card error states

image

### CODE REVIEW

File: packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-status/index.ts [hunk 1]
```diff
@@ -16,21 +16,26 @@ class CardPayWithdrawalWorkflowTransactionStatusComponent extends Component<Work
...
+      let result = yield this.layer2Network.awaitBridgedToLayer1(
+        this.layer2BlockHeightBeforeBridging!,
+        this.args.workflowSession.state.relayTokensTxnHash
+      );
+      this.args.workflowSession.update('bridgeValidationResult', result);
+      this.layer2Network.refreshBalances();
+      this.completedCount = 2;
+      this.args.onComplete?.();
+    } catch (e) {
+      this.error = true;
```
Comment (lukemelia) [inline packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-status/index.ts:37]: It seems bad to blackhole the error object here. Perhaps we should log it to sentry?

**Review by lukemelia — CHANGES_REQUESTED** [review]

File: packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-status/index.ts [hunk 2]
```diff
@@ -34,6 +34,9 @@ class CardPayWithdrawalWorkflowTransactionStatusComponent extends Component<Work
       this.completedCount = 2;
       this.args.onComplete?.();
     } catch (e) {
+      console.error('Failed to complete bridging to layer 1');
+      console.error(e);
+      Sentry.captureException(e);
```
Comment (Aierie) [inline packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-status/index.ts:39]: Does this look correct to you, @lukemelia? My understanding is Sentry doesn't automatically capture browser logs and requires manual capturing of caught exceptions.

Comment (Aierie) [reply packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-status/index.ts:39]: Based on: https://docs.sentry.io/platforms/javascript/guides/ember/usage/

Comment (lukemelia) [reply packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-status/index.ts:39]: I believe that console messages are captured as breadcrumbs but would not be treated as an error, so the approach you have here is good.

**Review by lukemelia — APPROVED** [review]

**Pull request merged**